### ISSUE TITLE
Overlay clean-up

### ISSUE THREAD

**Issue by xmhd** [adjust/gentoo-overlay#536]
## Overlay clean-up

tldr; there are many packages in the overlay that require a version bump and/or an EAPI bump. Or upstreamed to Gentoo. This issue will serve as a tracker for that work.

#### app-admin
- [ ] bagger-tools:
  - EAPI 5
  - PV: current
- [ ] consul-bin:
  - EAPI 7
  - PV: old
- [ ] goem:
  - EAPI 5
  - PV: live (9999) - TODO: tag a release and add keyworded ebuild
- [ ] kolide-k2-bin:
  - EAPI 7
  - PV: live (9999) - TODO: tag a release and add keyworded ebuild
- [ ] logstash-bin:
  - EAPI 5
  - PV: old / do we need newer available for testing?
- [ ] schaufel:
  - EAPI 7
  - PV: current
- [ ] tinderbox:
  - EAPI 6
  - PV: live (9999) - TODO: tag release and add keyworded ebuild
- [ ] trex:
  - Currently being unforked with patches sent upstream

#### app-eselect
- [ ] eselect-qtgraphicssystem:
  - EAPI 4
  - do we still need this?

#### app-metrics
- [ ] statsd_explorer:
  - EAPI 7
  - PV: old - do we need newer available for testing?

#### app-misc
- [ ] elasticsearch:
  - EAPI 5
  - PV: old - do we need newer available for testing?
- [ ] go-carbon:
  - EAPI 7
  - PV: old (new minor)
- [ ] karmabot-bin:
  - EAPI 6
  - PV: current
- [ ] logstash:
  - EAPI 5
  - PV: old - do we need newer available for testing?
- [X] rbenv:
  - EAPI 7
  - PV: current
- [X] ruby-build:
  - EAPI 7
  - PV: current

#### dev-cpp
- [ ] gtest:
  - EAPI 7
  - PV: old
  - Forked from Gentoo - do we hard depend on our old version or can we unfork and use upstream?

#### dev-db
- [ ] aerospike-server-enterprise:
  - EAPI 5
  - PV: 4.5.x / 4.6.x - do we need newer available for testing?
- [ ] ajbool:
  - EAPI 6
  - PV: current
- [ ] bagger-data:
  - EAPI 6
  - PV: current
- [ ] bagger-exports:
  - EAPI 6
  - PV: current
- [ ] bagger-master:
  - EAPI 6
  - PV: current
- [ ] base36:
  - EAPI 6
  - PV: current
- [ ] clickhouse:
  - EAPI 6
  - PV: old - new major available / do we need newer available for testing?
- [ ]  country:
  - EAPI 6
  - PV: current 
- [ ]  currency:
  - EAPI 6
  - PV: current
- [ ] device_type:
  - EAPI 6
  - PV: current
- [ ] geoip2lookup:
  - EAPI 6
  - PV: current
- [ ] hashtypes:
  - EAPI 6
  - PV: current
- [ ] istore:
  - EAPI 6
  - PV: current
- [ ] language:
  - EAPI 6
  - PV: current
- [ ] metabase-bin:
  - EAPI 6
  - PV: old
- [ ] metabase-clickhouse-driver-bin:
  - EAPI 6 
  - PV: old
- [ ] os_name:
  - EAPI 6
  - PV: current
- [X] parquet_fdw:
  - EAPI 7
  - PV: current
- [ ] pgbackrest:
  - EAPI 7
  - PV: old (new stable ver)
- [ ] pglogical:
  - EAPI 6
  - PV: old
- [ ] pgtelemetry :
  - EAPI 6
  - PV: current
- [ ] pipelinedb:
  - EAPI 6 
  - PV: current
- [X] postgresql: #533 
  - EAPI 7
  - PV: current
- [ ] roleman:
  - EAPI 6
  - PV: current
- [ ] timescaledb:
  - EAPI 6 
  - PV: old (new major) / do we need newew available for testing?

#### dev-go
- [ ] centrifugo:
  - EAPI 7
  - PV: old (new minor)
- [ ] golangci-lint-bin:
  - EAPI 7
  - PV: old (new minor)

#### dev-java
- [ ] icedtea{-bin}: 
  - feature request opened TODO_LINK

#### dev-lang
- [ ] elixir
  - EAPI 6
  - PV: old
- [ ] python: EOL
  - do we require this?
- [ ] R: forked from Gentoo to remove a dependency
  - do we require this? or can it be upstreamed to Gentoo
  - EAPI 6

#### dev-libs
- [X] apache-arrow:
  - EAPI 7
  - PV: current
- [ ] libaerospike:
  - EAPI 5
  - should this version match our aerospike-server-enterprise?
- [ ] thrift:
  - EAPI 7
  - PV: old (new minor)

#### dev-perl

- [ ] Any-URI-Escape:
  - EAPI 5
  - PV: current
- [ ] App-Sqitch :
  - EAPI 6
  - PV: old - new major
- [ ] Clone-PP:
  - EAPI 5
  - PV: old (new minor)
- [ ] Config-GitLike:
  - EAPI 6
  - PV: old (new minor)
- [ ] Data-IEEE754:
  - EAPI 5
  - PV: current
- [ ] Data-Printer:
  - EAPI 5
  - PV: old (new major)
- [ ] File-Rsync:
  - EAPI 5
  - PV: old
- [ ] Log-Any-Adapter-Callbacl:
  - EAPI 5
  - PV: current
- [ ] Math-Int128:
  - EAPI 5
  - PV: current
- [ ] Math-Int64:
  - EAPI 5
  - PV: current
- [ ] MaxMind-DB-Common:
  - EAPI 5
  - PV: current
- [ ] MaxMind-DB-Reader:
  - EAPI 5
  - PV: old (new minor)
- [ ] MaxMind-DB-Reader-XS:
  - EAPI 5
  - PV: old (new minor)
- [ ] MooX-StrictConstructor:
  - EAPI 5
  - PV: old (new minor)
- [ ] Net-Works:
  - EAPI 5
  - PV: current
- [ ] PGObject-Util-AsyncPool:
  - EAPI 5
  - PV: current
- [ ] PGObject-Util-PGConfig:
  - EAPI 5
  - PV: current
- [ ] PGObject-Util-Replication-Master:
  - EAPI 5
  - PV: current
- [ ] PGObject-Util-Replication-SMO:
  - EAPI 5
  - PV: current
- [ ] PGObject-Util-Replication-Slot:
  - EAPI 5
  - PV: current
- [ ] PGOBject-Util-Replication-Standby:
  - EAPI 5
  - PV: current
- [ ] Pod-Coverage-TrustPod:
  - EAPI 5
  - PV: old (new minor)
- [X] PostgresTools: #535 
  - EAPI 7
  - PV: current
- [ ] Proc-PID-File:
  - EAPI 5
  - PV: old (new minor)
- [ ] Proc-Queue:
  - EAPI 6
  - PV: current
- [ ] RedisFailover:
  - EAPI 5
  - PV: live (9999) TODO: tag a release and add keyworded ebuild
- [ ] RubyManager:
  - EAPI 5
  - PV: live (9999) TODO: tag a release and add keyworded ebuild
- [ ] Search-Elasticsearch-Client:
  - EAPI 5
  - PV: current
- [ ] Search-Elasticsearch:
  - EAPI 5
  - PV: old (new major)
- [ ] Statistics-Lite:
  - EAPI 5
  - PV: old (new major)
- [ ] Test-Bits:
  - EAPI 5
  - PV: current
- [ ] URI-Nested:
  - EAPI 5
  - PV: current
- [ ] URI-db:
  - EAPI 5
  - PV: old (new minor) 
- [ ]  Zabbix-tiny:
  - EAPI 5
  - PV: current

#### dev-python
- [ ] Authlib:
  - EAPI 6
  - PV: old
- [ ] EasyProcess:
  - EAPI 7
  - PV: old
- [ ] PyBambooHR:
  - EAPI 6
  - PV: old (new minor) - do we still need this now we are no longer using Bamboo (migrated to Workday)?
- [X] PyVirtualDisplay:
  - EAPI 7
  - PV: current
- [ ] carbonate:
  - EAPI 5
  - PV: old (new minor)
- [X] confluent-kafka:
  - EAPI 7
  - PV: old
- [ ] croniter:
  - EAPI 6
  - PV: old (new major)
- [ ] datadog:
  - EAPI 6
  - PV: old - do we require newer?
- [X] graphyte:
  - EAPI 7
  - PV: current
- [X] ipaddress:
  - EAPI 7
  - PV: current
- [X] marshmallow-sqlalchemy:
  - EAPI 7
  - old (new minor)
- [X] newrelic:
  - EAPI 7
  - PV: old (new major)
- [X] ns1-python:
  - EAPI 7
  - PV: current
- [ ] pagerduty-python-api-client:
  - EAPI 6
  - PV: live (9999) TODO: create keyworded ebuild pinned to a specific commit / date
- [ ] patroni:
  - EAPI 6
  - PV: old (new major)
- [X] pprintpp:
  - EAPI 7
  - PV: current
- [X] pycountry-convert:
  - EAPI 7
  - PV: current
- [ ] python-consul:
  - EAPI 6
  - PV: current
- [X] python-dotenv:
  - EAPI 7
  - PV: old
- [X] pyzabbix:
  - EAPI 7
  - PV: old
- [ ] rq-scheduler:
  - EAPI 6
  - PV: old
- [ ] rq:
  - EAPI 6
  - PV: old
- simple-salesforce:
  - EAPI 6

#### dev-qt
- [X] qtchooser:
  - EAPI7
  - deps match QT:4 - do we still need this?

#### dev-ruby
- [ ] fluent-plugin-postgres:
  - EAPI6
  - PV: old

#### dev-util
- [ ] android-ndk:
  - EAPI 6
  - PV: old - do we require new for testing?
- [ ] ephemeralpg:
  - EAPI 5
  - PV: old

#### net-analyzer
- [ ] clickhouse_exporter:
  - EAPI 6
  - PV: live (9999) - needs a release tagged
- [X] logdna-agent
  - EAPI 7
  - PV: old - do we require newer for testing?
- [ ] luigi:
  - EAPI 6
  - PV: current
- [X] nmis:
  - EAPI 7
  - PV: current
- [ ] prometheus-postgres_exporter:
  - EAPI 6
  - PV: current
- [ ] postgresql_exporter:
  - EAPI 6
  - PV: current
- [ ] prometheus:
  - EAPI 5
  - PV: old - do we need newer available for testing?
- [ ] rrdtool:
  - EAPI 7
  - PV: old
- [X] zabbix:
  - EAPI 7 
  - PV: old - do we need newer available for testing?

#### net-dns
- [X] octodns:
  - EAPI 7
  - PV: old (new minor v available)

#### net-proxy
- [ ] nutcrackers:
  - EAPI 5
  - PV: old - do we need newer available for testing?

#### sys-block
- [ ] hpssacli:
  - forked from Gentoo
  - PV: current
  - our PV is newer than Gentoo
  - can we maintain / proxy maintain this upstream?

#### sys-cluster
- [X] ceph:
  - forked from Gentoo to rescue old version
  - EAPI 7
- [X] chronos-bin:
  - EAPI 7
  - PV: current
- [X] flink-bin:
  - EAPI 7
  - PV: old - do we need newer available for testing?
  - note: newer requires Java 11
- [X] hadoop-bin:
  - EAPI 7
  - PV: current
- [X] hive-standalone-metastore-bin:
  - EAPI 7
  - PV: current
- [ ] mesos
  - EAPI 7
  - PV: old (new minor)
- [X] nomad-bin:
  - EAPI 7
  - PV: 0.x - do we need newer (1.x) available for testing?
- [X] spark-bin:
  - EAPI 7
  - PV:  2.4.x - do we need newer (3.x.x) available for testing?
- [ ] spark-nomad-bin:
  - EAPI 6
  - PV: current
  - note: DEPRECATED

#### sys-libs
- [ ] readline-compat
  - forked / rescued from Gentoo
  - do we still require this?

#### sys-kernel
- [ ] buildkernel:
  - EAPI 5
- [X] gentoo-kernel: 
  - EAPI 7
  - PV: old - TODO: add latest LTS 5.10
- [ ] gentoo-sources:
  - EAPI 6
  - PV: old - TODO: add latest LTS 5.10

#### sys-process
- [ ] vixie-cron:
  - not really maintained anymore
  - can we migrate to cronie? #4123 @ infrastructure
  - EAPI 0

#### virtual
- [ ] cron
  - EAPI 6
  - can be purged if we migrate to cronie

#### www-apps
- [ ] chromedriver-bin:
  - Forked 2 years ago
  - EAPI 6
  - PV: old
  - Do we still need this? it is upstream at Gentoo
- [ ] kibana-bin:
  - EAPI 7
  - PV: old - do we need newer available for testing?
- [ ] shiny-server:
  - EAPI 6
  - PV: old -  can we bump PV / do we need newer available for testing?

**Comment by xmhd** [comment]
#559 will be part of overlay clean-up
